README.md

```diff
@@ -2,15 +2,15 @@
 Visualizer of Sorting Algorithms made using Pure Vanilla JavaScript.
 
 ## Introduction
-It is a project to help learn shorting algorithms by implementing the algorithms but also tweak with the algorithm so that you can perform the animations required to visualize the algorithms
-
-It have bunch of li elements whose length is represented by the value of array and we need to implement shorting algorithm to arrange the li elements in shorted fashion
+This project creates animated visualizations in order to teach users how various sorting algorithms work. Users can select from several sorting algorithms and watch them run in real time (speed is adjustable). By watching how each algorithm sorts bars in ascending height, users gain a deep and intuitive understanding of the underlying logic of these algorithms and their relative efficiencies.
 
+## Implementation
+Sort functions are implemented in JavaScript. Bars of differing heights are represented by <li> elements and derive their height values from an array. Sorting algorithms direct re-rendering of bar elements until they appear in sorted order, thereby illustrating the sorting process.
```

Comment:
<li> would render bullet so its better to use &lt; or &gt; which will render < and > respectively and escape rendering html tags